.babelrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+    env: {
+        pure: {},
+        browser: {
+            presets: [
+                ["@babel/preset-env", {
+                    useBuiltIns: "usage",
+                    modules: false,
+                    corejs: {
+                        version: 3
+                    },
+                    targets: [
+                        "last 2 Chrome versions",
+                        "last 2 firefox versions",
+                        "last 2 Edge versions",
+                        "ie 10-11"
+                    ],
+                    // debug: true,
+                    loose: true, // needed for IE 10
+                }]
+            ]
+        },
+        development: {
+            presets: [
+                ["@babel/preset-env", {
+                    useBuiltIns: "usage",
+                    modules: "commonjs",
+                    corejs: {
+                        version: 3
+                    },
+                    targets: "maintained node versions"
```

Comment:
This will polyfill many things (even promises) but we specify in `package.json` and in docs that Node should be >= 10. I know that Node 10 does support everything that we use and we shouldn't even need `useBuiltIns`, but just in case, I would use `targets: "node 10"` here.